src/Azure.Functions.Cli/StaticResources/funcignore

```diff
@@ -5,3 +5,8 @@
 local.settings.json
 test
 getting_started.md
+node_modules/.bin/
+node_modules/@azure/
+node_modules/@types/
+node_modules/azure-functions-core-tools/
```

Comment:
I think this is the most impactful line, and I think it makes total sense to add it. Thank you!!!

Fwiw, adding to the funcignore will help with one-off deployments from core tools. However, if you deploy through a CI/CD pipeline I would recommend adding `npm prune --production` before deploying, which will more consistently get rid of _all_ dev dependencies.

Another fwiw, VS Code will run `npm prune --production` by default if you deploy through there.

Reply:
`npm prune` is a good shout, I was just trying to do a quick-n-dirty functionapp in this case 😉